packages/utils/src/stacktrace.ts

```diff
@@ -30,27 +30,17 @@ export function createStackParser(...parsers: StackLineParser[]): StackParser {
   };
 }
 
-interface StackParserOptions {
-  stackParser?: StackParser | StackLineParser[];
-}
-
 /**
- * Gets a stack parser implementation from options
+ * Gets a stack parser implementation from Options.stackParser
+ * @see Options
  *
  * If options contains an array of line parsers, it is converted into a parser
  */
-export function stackParserFromOptions(options: StackParserOptions | undefined): StackParser {
-  if (options) {
-    if (Array.isArray(options.stackParser)) {
-      options.stackParser = createStackParser(...options.stackParser);
-    }
-
-    if (typeof options.stackParser === 'function') {
-      return options.stackParser;
-    }
+export function stackParserFromOptions(stackParser: StackParser | StackLineParser[]): StackParser {
```

Comment:
After this change, seems like it's not being gotten from `options` anymore. Can we change the name to reflect that?

Reply:
This is a good point, and has got me thinking if we even should expose passing in a `StackLineParser[]`, and enforce that it is always an (optional) `StackParser`.

Reply:
The only plus to allow passing of `StackLineParser[]` is that users can configure stack parsing without needing to import and use `createStackParser` from `@sentry/utils`.

Some common lints warn if you use modules that are not direct dependencies so users would also need to add `@sentry/utils` to their depndencies or we would need to be re-export it from every sdk.